docs/index.md

```diff
@@ -51,3 +44,17 @@ Additional documentation on the concepts that support the operation of the main
 * [ipc](details/ipc.md): How all the components of Santa communicate.
   duction/syncing-overview.
 * [logs](details/logs.md): What and where Santa logs.
+
+### Binaries
+
+There are five main components that make up Santa, see the following pages to understand their functionality:
```

Comment:
This says five but links to four. In reality there are 6 (santasyncservice and santametricservice) but we don't have docs for those yet.